content/en/docs/contribute/Localization/Contribute_in_localization.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Contributing to an Existing Localization
+content_type: concept
+approvers:
+- remyleone
+- rlenferink
+weight: 50
+card:
+  name: contribute
+  weight: 50
+  title: Localizing the docs
+---
+
+<!-- overview -->
+
+This page shows you how to contribute in an existing
+
+[localization](https://blog.mozilla.org/l10n/2011/12/14/i18n-vs-l10n-whats-the-diff/).
+You can help add or improve the content of an existing localization. In
+[Kubernetes Slack](https://slack.k8s.io/), you can find a channel for each
+localization. There is also a general
+[SIG Docs Localizations Slack channel](https://kubernetes.slack.com/messages/sig-docs-localizations)
+where you can say hello.
+
+{{< note >}}
+For extra details on how to contribute to a specific localization,
+look for a localized version of this page.
+{{< /note >}}
+
+
+<!-- body -->
+
+### Find your two-letter language code
+
+First, consult the

```

Comment:
```suggestion
You can consult the

```

Reply:
As there is no second point in this section, rewording it to statement will be better.